src/printer.js

```diff
@@ -1790,6 +1790,10 @@ function printFunctionParams(path, print, options) {
     printed.push(concat([ "...", path.call(print, "rest") ]));
   }
 
+  if (printed.length === 0) {
+    return "()";
+  }
+
   return concat([
     "(",
     indent(
```

Comment:
I also tried adding `printed.length > 0 &&` next to `options.trailingComma`, which is the correct, fix but it doesn't look really good.

```js
veryLongCall(VERY_VERY_VERY_VERY_VERY_VERY_VERY_VERY_VERY_VERY_LONG_CONSTANT, (

) =>
  {});
```